gpflow/models/sgpr.py

```diff
@@ -346,33 +347,33 @@ def log_likelihood(self):
 
         return mahalanobisTerm + logNormalizingTerm * self.num_latent_gps
 
-    def predict_f(self, X: tf.Tensor, full_cov=False, full_output_cov=False) -> MeanAndVariance:
+    def predict_f(self, Xnew: InputData, full_cov=False, full_output_cov=False) -> MeanAndVariance:
         """
         Compute the mean and variance of the latent function at some new points
         Xnew.
         """
         _, _, Luu, L, _, _, gamma = self.common_terms()
-        Kus = Kuf(self.inducing_variable, self.kernel, X)  # size  [M, X]new
+        Kus = Kuf(self.inducing_variable, self.kernel, Xnew)  # [M, N]
```

Comment:
question: are M, N defined somewhere? Maybe in docstring point to that place. For new user these look a bit mysterious

Reply:
In GLOSSARY.md